docs/source/performance/caching.md

```diff
@@ -198,6 +199,81 @@ const server = new ApolloServer({
 }));
 ```
 
+### Recommended starting usage
+
+You usually don't need to specify cache hints for every field in your schema. Instead, we recommend doing the following as a starting point:
+
+* For fields that should _never_ be cached, explicitly set `maxAge` to `0`.
+
+* Set a `maxAge` for every field with a resolver that _actually fetches data from a data source_ (such as a database or REST API). You can base the value of `maxAge` on the frequency of updates that are made to the relevant data.
+
+* Set `inheritMaxAge: true` for each other field that returns a non-scalar type.
```

Comment:
This might make people think that this will be effective on root fields, but it won't be.